Emby.Server.Implementations/Emby.Server.Implementations.csproj

```diff
@@ -50,7 +51,7 @@
   </ItemGroup>
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.1</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
I don't think this is necessary, because this project doesn't need a reference to `Jellyfin.Server.Implementations`. The new services registered in `ApplicationHost` can be moved to `CoreAppHost` and then the reference to `Jellyfin.Server.Implementations` isn't necessary and this project can stay at `netstandard2.1`